docs/src/pages/customization/default-theme/default-theme.md

```diff
@@ -4,13 +4,13 @@
 
 ## Explore
 
-Explore the documentation theme object:
+Explore the default theme object:
```

Comment:
I have an impression of déjà vu, I wonder if we didn't iterate on this already in the past. If we did, it would be interesting to see the discussion we had.

Reply:
I had the same feeling (and had a quick look at git blame to try to find a related PR/issue, without any luck). As it stands, we show the default theme, so that's what we should document in the markdown.

Reply:
2 years ago: #9857. It seems that we still didn't find a good approach.

Reply:
Good find. Option 2 from the previous list - aligning the default theme and docs colors is the next move for v5, as already tentatively agreed.

Reply:
Ok, so we move forward like this?

Reply:
What about we change the default theme in a minor?

Reply:
It's a *relatively* low impact change – it won't actually break anything, and if a site isn't customising even the theme basics, it might not matter much to them; but IMHO it would break expectations to change this in a minor – to me, a visually breaking change is still a breaking change. 

Combined with the fact that this has been a low priority for a long time without any significant pushback, I suggest that we put it on hold, but with a clear plan to address it in v5.

Reply:
Sure